build.gradle.kts

```diff
@@ -64,8 +65,22 @@ subprojects {
     configurations.all {
         resolutionStrategy {
             dependencySubstitution {
-                substitute(module("com.facebook.react:react-native"))
-                    .with(module("com.facebook.react:react-native:$reactNativeVersion"))
+                if ( reactNativeMinorVersion >= 71 ) {
+                    // This substitution is based on React Native Gradle plugin.
+                    // Reference: https://t.ly/38jk
+                    substitute(module("com.facebook.react:react-android"))
+                        .with(module("com.facebook.react:react-android:$reactNativeVersion"))
```

Comment:
I think due to this `react-native` to `react-android` change, we need to stop using our own mirror from [react-native-mirror-publisher](https://github.com/wordpress-mobile/react-native-mirror-publisher). I think we can start using the artifacts published to `mavenCentral` and archive the `react-native-mirror-publisher` project.

If we keep using our own mirror, we might get duplicate class errors. Even if we don't, it wouldn't be great to have dependencies on both `react-android` & `react-native-mirror`.

The same goes for the `hermes-android`, of course.

@fluiddot What do you think?

P.S: If you all decide to go through with this ^ suggestion, please let me know if you need any help!

Reply:
That's great to hear!